uefi/biosregion.go

```diff
@@ -5,10 +5,14 @@ import (
 	"strings"
 )
 
+// BiosRegion represents the Bios Region in the firmware.
+// It holds all the FVs as well as padding
+// TODO(ganshun): handle padding
```

Comment:
nit: I normally avoid "assigning" a TODO because it often discourages other people from doing it. However I'm happy to hear if you have stronger reasons in favour!

Reply:
I see! I guess I'll file issues instead next time